kobo/apps/help/serializers.py

```diff
@@ -0,0 +1,71 @@
+from django.db import transaction
```

Comment:
Same comments on imports

Reply:
To me, the imports here are correct because they're broken into the PEP-8 sections and ordered within those sections by line length.